### PR TITLE
feat: [PAGOPA-1066] update iban crud: added description to iban_master

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- added field description to iban_master table

#### Motivation and Context

- different descriptions for the same iban

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

**Comment by alessio-acitelli** [comment]
update_code

File: pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   <groupId>it.gov.pagopa.api-config</groupId>
   <artifactId>starter</artifactId>
-  <version>1.8.0</version>
+  <version>1.9.0</version>
```
Comment (andrea-deri) [inline pom.xml:16]: why the manual versioning?

Comment (alessio-acitelli) [reply pom.xml:16]: I didn't make the change, I think it's automatic

**Review by andrea-deri — APPROVED** [review]
good job!

**Pull request merged**